README.rst

```diff
@@ -2,47 +2,70 @@
 pygeobase
 =========
 
+.. image:: https://travis-ci.org/TUW-GEO/pygeobase.svg?branch=master
+    :target: https://travis-ci.org/TUW-GEO/pygeobase
+
+.. image:: https://coveralls.io/repos/github/TUW-GEO/pygeobase/badge.svg?branch=master
+   :target: https://coveralls.io/github/TUW-GEO/pygeobase?branch=master
+
+.. image:: https://badge.fury.io/py/pygeobase.svg
+    :target: https://badge.fury.io/py/pygeobase
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1164877.svg
+   :target: https://doi.org/10.5281/zenodo.1164877
```

Comment:
We should probably cite doi https://doi.org/10.5281/zenodo.846761 here since it will always forward to the latest version of the package. This is not yet consistent accross all packages but we best start with it.

Reply:
I just discussed this with @mazweg and the question we asked ourselves was whether it is of interest to know which version a user installed/used in the end? What do you think?

Reply:
BTW if you @cpaulik have some comments regarding harmonization between the code packages please let us know. @mazweg is about to create a (more or less) common README and fix all failing readthedocs issues.

Reply:
If somebody cites it correctly then they should use the DOI of the exact version that they used. But since this is the link in the README which people will click it should probably be linked to the over project DOI. Otherwise we would have to update the link of the batch after each release which is not going to happen realistically.

I'll be looking over the PR's but from what I've seen so far it looks good.

Reply:
If we want to give more information about the citation we could also link to http://help.zenodo.org/#versioning

Reply:
OK, I agree to add the badge of the "common/latest" DOI, but also add more information and a link to the DOIs of the specific versions. @mazweg can make a draft for pygeobase and adapt it later for the other packages.